plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -1413,7 +1411,7 @@ public RowChangeParadigm getRowChangeParadigm(ConnectorSession session, Connecto
     @Override
     public ColumnHandle getMergeRowIdColumnHandle(ConnectorSession session, ConnectorTableHandle tableHandle)
     {
-        return new DeltaLakeColumnHandle(ROW_ID_COLUMN_NAME, MERGE_ROW_ID_TYPE, OptionalInt.empty(), ROW_ID_COLUMN_NAME, MERGE_ROW_ID_TYPE, SYNTHESIZED);
+        return DeltaLakeColumnHandle.getMergeRowIdColumnHandle();
```

Comment:
static import pls

Reply:
Since `getMergeRowIdColumnHandle` method is already present in `DeltaLakeMetadata`, so can't call directly `getMergeRowIdColumnHandle`.

Compiler throws error:

```
'getMergeRowIdColumnHandle(io.trino.spi.connector.ConnectorSession, io.trino.spi.connector.ConnectorTableHandle)' in 'io.trino.plugin.deltalake.DeltaLakeMetadata' cannot be applied to '()'
```